autoware_launch/launch/autoware.launch.xml

```diff
@@ -91,6 +91,7 @@
   <!-- Localization -->
   <group if="$(var launch_localization)">
     <include file="$(find-pkg-share tier4_localization_launch)/launch/localization.launch.xml"/>
+    <include file="$(find-pkg-share default_ad_api_helpers)/launch/auto_pose_init.launch.xml"/>
```

Comment:
Can you add this into localization.launch.xml?

Reply:
This node just calls the API periodically. The API initializes the pose with GNSS when no pose is specified. So I think this node is an API utility rather than localization. Can I put it in localization?

Reply:
I think it is good.
I want to keep it as simple as possible in autoware_launch.
@mitsudome-r how about you?

Reply:
Move this line.
https://github.com/autowarefoundation/autoware.universe/pull/1431/commits/8c987eb54e3bcda22c1df08d2cc34e1111ac8066

Reply:
@isamu-takagi Thank you! Is this PR closed?